src/main/scala/firrtl/passes/ReplSeqMem.scala

```diff
@@ -88,7 +88,7 @@ Optional Arguments:
     error("No circuit name specified for ReplSeqMem!" + usage)
   )
   val target = CircuitName(passCircuit)
-  def duplicate(n: Named) = this copy (t = (t replace (s"-c:$passCircuit", s"-c:${n.name}")))
+  def duplicate(n: Named) = this copy (t = t replace(s"-c:$passCircuit", s"-c:${n.name}"))
```

Comment:
Similarly, copy should 'look more like a function' than replace since it uses named args.

I generally advocate for preferring using . and no space rather than spaces except in the rare cases where it is a short chaining of 'common binary operator-like functions' (like map, reduce, fold, and corollaries). Really, it is more of a: 'when in doubt, use function calls'.

Note that, in the case where you need to split things across multiple lines, going back to the function call notation is fairly helpful....
```scala
val myResult: Seq[Int] =
  seqInts
  .map(x => x - (x % 2))  // round down to nearest even
  .scanLeft(0)(_+_)  // take running sum of numbers
  .drop(1)  // drop the first sum
  .filter(_ % 2 == 0) // filter to only even sums
```